src/ape/utils.py

```diff
@@ -144,7 +144,7 @@ def load_config(path: Path, expand_envars=True, must_exist=False) -> Dict:
         return config or {}
 
     elif must_exist:
-        raise IOError(f"{path} does not exist!")
+        raise OSError(f"{path} does not exist!")
```

Comment:
IOError was removed from the exception hierarchy since python 3.3: https://docs.python.org/3/library/exceptions.html#exception-hierarchy

it is an alias for OSError.

Here is a GitHub action where this is is discussed: https://github.com/asottile/pyupgrade/issues/141

I kind of like `IOError` sometimes, but I am not sure here is one of those times...